polaris-react/src/components/Breadcrumbs/Breadcrumbs.tsx

```diff
@@ -16,6 +18,7 @@ export interface BreadcrumbsProps {
 
 export function Breadcrumbs({backAction}: BreadcrumbsProps) {
   const {content} = backAction;
+  const {polarisSummerEditions2023} = useFeatures();
 
   const contentMarkup = (
```

Comment:
probably missing context here but can we just get rid of this and just use the `Button` component?